src/XrdPfc/XrdPfc.cc

```diff
@@ -1071,6 +1075,10 @@ int Cache::Prepare(const char *curl, int oflags, mode_t mode)
    // Do not allow write access.
    if (oflags & (O_WRONLY | O_RDWR | O_APPEND | O_CREAT))
    {
+      if (Cache::GetInstance().RefConfiguration().m_write_through)
+      {
+         return 0;
```

Comment:
Returning a 0 will cause a SEGV. You need to return the XrdOucCacheIO object that was passed to you via the args so that you pass through the I/O.

Reply:
(This appears to be a duplicate comment so I am just resolving the conversation)